docker/install-dynamic-plugins.py

```diff
@@ -216,12 +216,13 @@ def main():
         print('\n======= Installing dynamic plugin', package, flush=True)
 
         package_is_local = package.startswith('./')
+
+        # If package is not local, then integrity check is mandatory
+        if not package_is_local and not skipIntegrityCheck and not 'integrity' in plugin:
```

Comment:
```suggestion
        if not (package_is_local and skipIntegrityCheck and 'integrity' in plugin):
```
sure this method works as well I guess. I personally felt the `if` `elif` chain was more concise and clear.